src/storage/mvcc/reader/point_getter.rs

```diff
@@ -75,6 +75,7 @@ impl<S: Snapshot> PointGetterBuilder<S> {
         let write_cursor = CursorBuilder::new(&self.snapshot, CF_WRITE)
             .fill_cache(self.fill_cache)
             .prefix_seek(!self.multi)
+            .scan_mode(ScanMode::Mixed)
```

Comment:
I think you should use ScanMode::Forward when the multi is false.

Reply:
Nice catch. It is more strict.

Reply:
Forward can prevent iterator unexpected back and forward moving.

Reply:
Updated, PTAL again.